src/FullBuild/Views.fs

```diff
@@ -83,8 +83,17 @@ with
         projects
 
     member this.Save (isDefault : bool option) =
-        let viewId = Anthology.ViewId this.View.Name
-        Configuration.SaveView viewId this.View isDefault
+        let viewId = Anthology.ViewId this.View.Name

```

Comment:
Is is possible to recover Configuration.SaveView and use it instead of inlining ?